mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/javadsl/MqttSession.scala

```diff
@@ -52,19 +51,19 @@ abstract class MqttClientSession extends MqttSession {
 }
 
 object ActorMqttClientSession {
-  def create(settings: MqttSessionSettings, mat: Materializer, system: ActorSystem): ActorMqttClientSession =
-    new ActorMqttClientSession(settings, mat, system)
+  def create(settings: MqttSessionSettings, system: ClassicActorSystemProvider): ActorMqttClientSession =
```

Comment:
Is that already in anticipation that `untyped.ActorSystem extends ClassActorSystemProvider` in 2.5 or does it already work with 2.5.30?

Reply:
You're right, I did this before I found out that it doesn't work as expected. https://github.com/akka/akka/issues/28764

I need to allow both `akka.actor.ActorSystem` and the provider.

Reply:
No. Good that I had lunch. This connector is Akka 2.6 only.

Reply:
Ah, that explains it!